File: Java/Easy/Reverse-String.java
class Solution {
  public void reverseString(char[] s) {
    char[] answer = Arrays.copyOf(s, s.length);
    for (int i = 0; i < s.length; i++) {
      s[i] = answer[s.length - 1 - i];
    }
  }
}

File: C++/Easy/Reverse-String.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        int len = s.size();
        int mid = len/2;
        for (int i = 0; i<mid; i++) {
            char temp = s[i];
            s[i] = s[len-i-1];
            s[len-i-1] = temp;
        }
    }
};
